src/System.Drawing.Common/src/System/Drawing/BufferedGraphics.Unix.cs

```diff
@@ -112,10 +90,9 @@ public void Render(Graphics target)
         }
 
         [MonoTODO("The targetDC parameter has no equivalent in libgdiplus.")]
-        public void Render(IntPtr targetDC)
+        private void RenderInternal(HandleRef refTargetDC)
         {
             throw new NotImplementedException();
```

Comment:
Again, for a subsequent change rather than this one, is NotImplementedException correct here, or should it be PlatformNotSupportedException?